lib/tree_stand/node.rb

```diff
@@ -131,6 +173,66 @@ def each(&block)
       enumerator
     end
 
+    # Enumerate named children.
+    # @example
+    #   node.text # => "3 * 4"
+    #
+    # @example Iterate over the child nodes
+    #   node.each do |child|
```

Comment:
Should this be `each_named`?

```suggestion
    #   node.each_named do |child|
```

Reply:
Yes